src/resources/OrganizationAccess/AccessInterfaces.ts

```diff
@@ -22,9 +22,32 @@ export interface AccessParams {
 }
 
 export interface AccessModel {
-    accessLevel: AccessLevel[];
-    callerPartOf: boolean;
+    /*
+     * The access level for this privilege.
+     */
+    accessLevel: AccessLevel;
+    /*
+     * Whether the calling user is part of this group.
+     */
+    callerPartOf?: boolean;
+    /*
+     * The creation date of this privilege.
+     */
+    createdDate?: string;
+    /*
+     * The display name of this privilege.
+     */
     displayName: string;
+    /*
+     * The id of this privilege.
+     */
     id: string;
-    privilegeholderType: PrivilegeholderType;
+    /*
+     * The type of this privilege.
+     */
+    privilegeHolderType: PrivilegeHolderType;
+    /*
+     * The list of sources id this privilege is applied to.
```

Comment:
AccessModel is not only used by the sources, but by all the resources with granular privileges (query pipelines, sources, groups, api keys, models, search pages, maybe others that I forget...)

```suggestion
     * The list of resources ids this access model has edit access level on.
```